src/main/resources/config/en.yml

```diff
@@ -8,6 +8,14 @@ ConfigVersion: ${version}
 # You must restart your server after changing this option
 BotToken: "BOTTOKEN"
 
+# Proxy settings, leave empty if you dont need this
+# You must restart your server after changing this option
+ProxyHost: ""
+ProxyPort: ""
```

Comment:
This option should be a number instead of a string

Reply:
Proxy host should could be set to a domain. I believe you mean the port setting should be a number?